src/config/normalize.ts

```diff
@@ -66,6 +66,15 @@ function getStorageDriverConfig(
 			driverConfig: storage.sqliteDriverConfig,
 			...extras,
 		}
+	} else if (storage.driverName === Redis) {
+		return {
+			driverName: Redis,
+			driverConfig: {
+				...storage.redisDriverConfig,
+				port: Number(storage.redisDriverConfig.port),
```

Comment:
```suggestion
				port: +storage.redisDriverConfig.port,
```

Reply:
I had to add casting.
Without it i'm getting a typescript error.